tests/languages/bash/process_substitution_feature.test

```diff
@@ -0,0 +1,55 @@
+<(yes) <(if true; then :; fi) <(echo) <(true)
+>(yes) >(if true; then :; fi) >(echo) >(true)
+
+----------------------------------------------------
+
+[
+	["operator", ["<"]],
+	["punctuation", "("],
+	["function", "yes"],
+	["punctuation", ")"],
+	["operator", ["<"]],
+	["punctuation", "("],
+	["keyword", "if"],
+	["boolean", "true"],
+	["punctuation", ";"],
+	["keyword", "then"],
+	["builtin", ":"],
+	["punctuation", ";"],
+	["keyword", "fi"],
+	["punctuation", ")"],
+	["operator", ["<"]],
+	["punctuation", "("],
+	["builtin", "echo"],
+	["punctuation", ")"],
+	["operator", ["<"]],
+	["punctuation", "("],
+	["boolean", "true"],
+	["punctuation", ")"],
+	["operator", [">"]],
+	["punctuation", "("],
+	["function", "yes"],
+	["punctuation", ")"],
+	["operator", [">"]],
+	["punctuation", "("],
+	["keyword", "if"],
+	["boolean", "true"],
+	["punctuation", ";"],
+	["keyword", "then"],
+	["builtin", ":"],
+	["punctuation", ";"],
+	["keyword", "fi"],
+	["punctuation", ")"],
+	["operator", [">"]],
+	["punctuation", "("],
+	["builtin", "echo"],
+	["punctuation", ")"],
+	["operator", [">"]],
+	["punctuation", "("],
+	["boolean", "true"],
+	["punctuation", ")"]
+]
+
+----------------------------------------------------
+
+Checks for variables.

```

Comment:
I wonder if this test file is needed on its own. Maybe a file `operator_feature.test` could test all operators including process substitution?